src/Nordea/Components/TextInput.elm

```diff
@@ -89,25 +75,27 @@ getAttributes config =
         , config.placeholder |> Maybe.map placeholder
         ]
 
-
-
 -- STYLES
-
-
-styles : List Style
-styles =
+getStyles : Config msg -> List Style
+getStyles config =
+    let
+        borderColorStyle =
+            if config.showError then Colors.redDark
+            else Colors.grayMedium
+    in
     [ fontSize (rem 1)
-    , height (em 2.5)
-    , padding2 (em 0.75) (em 0.75)
-    , borderRadius (em 0.125)
-    , border3 (em 0.0625) solid Colors.grayMedium
-    , boxSizing borderBox
-    , width (pct 100)
-    , disabled
-        [ backgroundColor Colors.grayWarm
-        ]
-    , focus
-        [ outline none
-        , borderColor Colors.blueNordea
+        , height (em 2.5)
+        , padding2 (em 0.75) (em 0.75)
+        , borderRadius (em 0.125)
+        , border3 (em 0.0625) solid borderColorStyle
+        , boxSizing borderBox
+        , width (pct 100)
+        , disabled
+            [ backgroundColor Colors.grayWarm
+            ]
+        , focus
+            [ outline none
+            , borderColor Colors.blueNordea
+            ]
```

Comment:
Seems like the formatting is a bit off. Could you run elm-format again? (note: you have to enable elm-format for every project in IntelliJ IDEA 😉 )

Reply:
👀 for details! I'll fix!